renderer/components/langUtils.js

```diff
@@ -13,15 +11,15 @@ export const getMessages = (locale = null) => {
     supportedMessages = window.OONITranslations
   }
 
-  if (supportedMessages.hasOwnProperty(locale)) {
+  if (locale in supportedMessages) {
     const mergedMessages = Object.assign(
       {},
       supportedMessages[defaultLocale],
       supportedMessages[locale]
     )
     return mergedMessages
   } else {
```

Comment:
Maybe you can even drop the `else` statement here, to make it more readable and robust.